### PR TITLE
Adds a toggle + admin preference for who/adminwho admin messages

### PR DESCRIPTION
**Pull request by TTerc**
## About the PR 

Title. Who/adminwho will still be logged regardless
Also, alerts are turned on by default in admin preferences

## Why's this needed? 

It was discussed in adminchat and admins who participated in the talk seemed to be okay with the idea

### CODE REVIEW

**Comment by TTerc** [comment]
Don't merge yet, I'm making it a preference actually

File: code/mob/who.dm [hunk 1]
```diff
@@ -124,4 +124,5 @@
 	if(!usr.client.holder)
 		logTheThing("admin", usr, null, "used adminwho and saw [adwnum] admins.")
 		logTheThing("diary", usr, null, "used adminwho and saw [adwnum] admins.", "admin")
-		message_admins("<span class='internal'>[key_name(usr)] used adminwho and saw [adwnum] admins.</span>")
+		if(usr.client.holder.adminwho_alerts)
```
Comment (Tarmunora) [inline code/mob/who.dm:127]: Uh, I don't think that's gonna work. See the `if` block that it's in...

**Review by Tarmunora — CHANGES_REQUESTED** [review]

File: code/mob/who.dm (hunk 1 above)
Comment (TTerc) [reply code/mob/who.dm]: Fixed and tested on local server with 2 accounts connected, one of them being a regular player one, works fine now.

**Comment by TTerc** [comment]
Time to be shameless

**Pull request merged**